lib/effects/effects.dart

```diff
@@ -170,13 +169,13 @@ abstract class PositionComponentEffect
   /// another effect, like children of a CombinedEffect or SequenceEffect).
   void _setComponentState(Vector2 position, double angle, Vector2 size) {
     if (isRootEffect()) {
-      if (modifiesPosition) {
+      if (modifiesPosition && position != null) {
```

Comment:
Maybe we should add assert, instead of just silently doing nothing?

Reply:
> Maybe we should add assert, instead of just silently doing nothing?

yeah, this makes sense